bsp/esp_bsp_generic/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+file(GLOB_RECURSE SRCS src/*.c)
+
+#IDF version is less than IDF5.0
+if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_LESS "5.0")
+    #set(SRC_VER "esp_bsp_generic_idf4.c")
+    set(REQ_VER "led_strip")
```

Comment:
I'm sorry, I could not quickly find reason for this... Could you please explain?

Reply:
I am sorry, I forgot remove it after some changes, which I made before. Removed.